### PR TITLE
test(cmd-api-server): jestify grpc-proto-gen-ts-client-healthcheck test

### PR DESCRIPTION
**Pull request by petermetz**
1. Also fixing a minor bug in config/config-service-example-config-validity.test.ts
where the path of the temporary files generated by the test were being saved
in the wrong directory instead of $PROJECT_ROOT/.tmp/...

Signed-off-by: Peter Somogyvari <peter.somogyvari@accenture.com>

**Pull Request Requirements**
- [x] Rebased onto `upstream/main` branch and squashed into single commit to help maintainers review it more efficient and to avoid spaghetti git commit graphs that obfuscate which commit did exactly what change, when and, why.
- [x] Have git sign off at the end of commit message to avoid being marked red. You can add `-s` flag when using `git commit` command. You may refer to this [link](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) for more information.
- [x] Follow the Commit Linting specification. You may refer to this [link](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#specification) for more information. 

**Character Limit**
- [x] Pull Request Title and Commit Subject must not exceed 72 characters (including spaces and special characters).
- [x] Commit Message per line must not exceed 80 characters (including spaces and special characters).

**A Must Read for Beginners**
For rebasing and squashing, here's a [must read guide](https://github.com/servo/servo/wiki/Beginner's-guide-to-rebasing-and-squashing) for beginners.

### CODE REVIEW

File: packages/cactus-cmd-api-server/src/test/typescript/unit/grpc-proto-gen-ts-client-healthcheck.test.ts [hunk 1]
```diff
@@ -9,80 +9,95 @@ import { AuthorizationProtocol } from "../../../main/typescript/public-api";
...
+    // const res = await responsePromise;
+
+    // const resHc = res?.toObject();
+
+    // t.ok(resHc, `healthcheck response truthy OK`);
+    // t.ok(resHc?.createdAt, `resHc.createdAt truthy OK`);
+    // t.ok(resHc?.memoryUsage, `resHc.memoryUsage truthy OK`);
+    // t.ok(resHc?.memoryUsage?.rss, `resHc.memoryUsage.rss truthy OK`);
+    // t.ok(resHc?.success, `resHc.success truthy OK`);
+    // t.end();
```
Comment (outSH) [inline packages/cactus-cmd-api-server/src/test/typescript/unit/grpc-proto-gen-ts-client-healthcheck.test.ts:101]: Can this commented code be removed?

**Review by outSH — CHANGES_REQUESTED** [review]
Thanks, left one small comment

File: packages/cactus-cmd-api-server/src/test/typescript/unit/grpc-proto-gen-ts-client-healthcheck.test.ts (hunk 1 above)
Comment (petermetz) [reply packages/cactus-cmd-api-server/src/test/typescript/unit/grpc-proto-gen-ts-client-healthcheck.test.ts:101]: @outSH Oops, thank you for catching that!

**Review by jagpreetsinghsasan — APPROVED** [review]

**Review by outSH — APPROVED** [review]
Thanks!

**Pull request merged**